dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3PutJobTest.java

```diff
@@ -44,34 +50,51 @@
 import org.mockito.Mockito;
 
 import java.io.File;
-import java.net.URL;
+import java.io.FileNotFoundException;
+import java.net.URISyntaxException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
+import java.util.ResourceBundle;
 import java.util.concurrent.CountDownLatch;
 
 import static org.junit.Assert.assertTrue;
 
 public class Ds3PutJobTest {
 
     private static final JobWorkers jobWorkers = new JobWorkers(10);
-    private final static String bucketName = SessionConstants.ALREADY_EXIST_BUCKET;
+    private final static String bucketName = "Ds3PutJobTestBucket";
+    private static final Ds3Client client = Ds3ClientBuilder.fromEnv().withHttps(false).build();
     private static Session session;
     private static File file;
     private static Ds3PutJob ds3PutJob;
     private boolean successFlag = false;
+    private final static ResourceBundle resourceBundle = ResourceBundle.getBundle("lang", new Locale(ConfigProperties.getInstance().getLanguage()));
+    private static final BuildInfoServiceImpl buildInfoService = new BuildInfoServiceImpl();
 
     @BeforeClass
     public static void setConnection() {
         new JFXPanel();
         Platform.runLater(() -> {
-            final SavedSession savedSession = new SavedSession(SessionConstants.SESSION_NAME, SessionConstants.SESSION_PATH, SessionConstants.PORT_NO, null, new SavedCredentials(SessionConstants.ACCESS_ID, SessionConstants.SECRET_KEY), false);
-            session = new CreateConnectionTask().createConnection(SessionModelService.setSessionModel(savedSession, false));
-            final ClassLoader classLoader = Ds3PutJobTest.class.getClassLoader();
-            final URL url = classLoader.getResource(SessionConstants.LOCAL_FOLDER + SessionConstants.LOCAL_FILE);
-            if (url != null) {
-                Ds3PutJobTest.file = new File(url.getFile());
+            final SavedSession savedSession = new SavedSession(
+                    "Ds3PutJobTest",
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = CreateConnectionTask.createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            final Path path;
+            try {
+                path = ResourceUtils.loadFileResource("files/SampleFiles.txt");
+                Ds3PutJobTest.file = path.toFile();
+            } catch (URISyntaxException | FileNotFoundException e) {
+                e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.